client/my-sites/plans/jetpack-plans/product-store/types.ts

```diff
@@ -42,11 +42,15 @@ export interface ViewFilterProps {
 }
 
 export type ProductsListProps = ProductStoreBaseProps &
-	Omit< ProductStoreProps, 'urlQueryArgs' | 'header' >;
+	Omit< ProductStoreProps, 'urlQueryArgs' | 'header' > & {
+		clickMoreHandlerFactory: ( item: SelectorProduct ) => VoidFunction | undefined;
+	};
 
 export type BundlesListProps = ProductsListProps;
 
-export interface ItemsListProps extends ProductsListProps {
+export type itemToDisplayProps = Omit< ProductsListProps, 'clickMoreHandlerFactory' >;
```

Comment:
Types should use PascalCase/TitleCase.

```suggestion
export type ItemToDisplayProps = Omit< ProductsListProps, 'clickMoreHandlerFactory' >;
```

Reply:
thanks for spotting this @manzoorwanijk .. this one is fix 👍